src/lib/statusEnums.ts

```diff
@@ -0,0 +1,9 @@
+export enum LoginStatus {
+    ALREADY_LOGGED_IN,
+    SUCCESSFUL_LOGIN,
+    SUCCESSFUL_LOGOUT
+}
+export enum RegisterStatus {
+    ALREADY_LOGGED_IN,
```

Comment:
Unterscheiden wir zwischen already logged in und logged in, weil dann entweder beide jetzt schon rein sollten oder nur logged in.

Reply:
Das Use-Case hier ist besser wenn es **ALREADY** heißt. Es ist nur für uns, damit wir wissen, welche Nachricht wann auf der Startseite angezeigt wird, die enums werden wir höchstwahrscheinlich nicht mehr nutzen, da der Rest was das angeht von Daniel gemacht wird! Da eine Fehlermeldung kommt, wenn der user **bereits** angemeldet ist. Sollte es  nun allerdings nur LOGGED_IN heißen, kann es auch heißen, dass der user sich angemeldet hat. Es soll aber klar werden, dass er weil er schon angemeldet **ist** auf dieses Route nicht kann. Deswegen das **ALREADY**